device-types/Ubiquiti/USW-Enterprise-48-PoE.yaml

```diff
@@ -0,0 +1,120 @@
+---
+manufacturer: Ubiquiti
+model: UniFi Switch Enterprise 48 PoE
+slug: unifi-switch-enterprise-48-poe
+part_number: USW-Enterprise-48-PoE
+comments: |
+  UniFi Switch Enterprise 48 PoE (150W + max. 720W PoE), (48) 2.5 Gigabit PoE+ IEEE 802.3af/at, (4) SFP+
+
+  Dimensions: 442 x 44 x 400 mm (17.4 x 1.7 x 15.7")
+u_height: 1
+is_full_depth: true
+interfaces:
+  - name: Port 1 (PoE)
```

Comment:
Please enter the PoE attributes instead of putting PoE in brackets

Reply:
sorry I forgot that's a feature now since I still have to migrate my old netbox deployment.
Hopefully okay now?